composer.json

```diff
@@ -165,7 +163,6 @@
         "toflar/cronjob-supervisor": "^2.0",
         "toflar/psr6-symfony-http-cache-store": "^4.0",
         "twig/extra-bundle": "^3.0",
-        "twig/string-extra": "^3.0",
```

Comment:
This was added in https://github.com/contao/contao/pull/5247 and is used here: https://github.com/contao/contao/blob/6692077fe222834ca5e071e844e4aacca707c1cc/core-bundle/contao/templates/twig/backend/template_skeleton.html.twig#L33